tests/fixtures/models/image-polymorphic-relationship.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Image extends Model
+{
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'url',
+        'imageable_id',
```

Comment:
Probably should not be fillable. However, I think the new syntax would solve this.